workspace/workspace-pref/package.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "@pnpm/workspace-pref",
```

maybe this could be better
```suggestion
  "name": "@pnpm/workspace.spec-parser",
```